rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -981,6 +981,95 @@ def test_staking_performance_division_by_zero_protection(eth2) -> None:
         assert FVal(validator_apr['apr']) == ZERO
 
 
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+@pytest.mark.parametrize('ethereum_accounts', [[make_evm_address()]])
+def test_accumulating_validator_exit_pnl_with_multiple_deposits(
+        eth2: 'Eth2',
+        database: 'DBHandler',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    """Test that exit PnL calculation for accumulating validators with multiple deposits is correct.
+
+    This test reproduces the issue where an accumulating validator (0x02 type) with:
+    - Initial 32 ETH deposit
+    - Multiple additional 8 ETH deposits (total 64 ETH deposited)
+    - Accumulated consensus rewards through skimming withdrawals
+    - Exits with a balance higher than deposits
+
+    The issue is that the exit PnL incorrectly counts the entire difference between
+    exit amount and deposits as profit, even though consensus rewards were already
+    accounted for in withdrawals.
```

Comment:
```suggestion
    The issue was that the exit PnL incorrectly counted the entire difference between
    exit amount and deposits as profit, even though consensus rewards were already
    accounted for in withdrawals.
```
This should be in past tense since its fixed now right? Maybe should also link the issue mentioning that its a regression test.